plutus-tx-plugin/test/Plugin/Data/recursive/interListConstruct.tplc-read.golden

```diff
@@ -0,0 +1,265 @@
+(program 1.1.0 ((/\(InterList_i0 :: * -> * -> *) ->
+                   \(InterCons_i0
+                       : all a_i0
+                             b_i0.
+                           a_i2 -> b_i1 -> InterList_i4
+                                             b_i1
+                                             a_i2 -> InterList_i4 a_i2 b_i1)
+                    (InterNil_i0 : all a_i0 b_i0. InterList_i5 a_i2 b_i1)
+                    (InterList_match_i0
+                       : all a_i0
+                             b_i0.
+                           InterList_i6
+                             a_i2
+                             b_i1 -> (all out_InterList_i0.
+                                        (a_i3 -> b_i2 -> InterList_i7
+                                                           b_i2
+                                                           a_i3 -> out_InterList_i1) -> out_InterList_i1 -> out_InterList_i1)) ->
+                     (/\Bool_i0 ->
+                        \(True_i0 : Bool_i2)
+                         (False_i0 : Bool_i3)
+                         (Bool_match_i0
+                            : Bool_i4 -> (all out_Bool_i0.
+                                            out_Bool_i1 -> out_Bool_i1 -> out_Bool_i1)) ->
+                          InterCons_i7
+                            {integer}
+                            {Bool_i4}
+                            0
+                            False_i2
+                            (InterCons_i7
+                               {Bool_i4}
+                               {integer}
+                               False_i2
+                               -1
+                               (InterCons_i7
+                                  {integer}
+                                  {Bool_i4}
+                                  42
+                                  True_i3
+                                  (InterNil_i6 {Bool_i4} {integer}))))
+                       {sop [] []}
+                       (constr (sop [] []) 0 [])
+                       (constr (sop [] []) 1 [])
+                       (\(x_i0 : sop [] []) ->
+                          /\out_Bool_i0 ->
+                            \(case_True_i0 : out_Bool_i2)
+                             (case_False_i0 : out_Bool_i3) ->
+                              case out_Bool_i3 x_i4 [ case_True_i2
+                                                    , case_False_i1 ]))
+                  {\a_i0
+                    b_i0 ->
+                     ifix (\(rec_i0 :: ((* -> * -> *) -> *) -> *)
+                            (spine_i0 :: (* -> * -> *) -> *) ->
+                             spine_i1
+                               ((\(InterList_i0 :: * -> * -> *)
+                                  a_i0
+                                  b_i0 ->
+                                   sop [ a_i2
```

Comment:
I know I started it, but I'm still unsure how I feel about how this looks like a single space indent here.

Reply:
I don't like it too.